### ISSUE TITLE
Annotation design caused user confusion

### ISSUE THREAD

**Issue by MyfanwyNixon** [mysociety/theyworkforyou#1130]
> What does the letter L in a purple square mean ?

note: I assume he means circle, as in:

![fireshot screen capture 757 - review on antimicrobial resistance_ 5 jul 2016_ house of commons debates - theyworkforyou - www_theyworkforyou_com_de](https://cloud.githubusercontent.com/assets/944656/16624360/43d16ab6-4399-11e6-91f4-814264219ef4.jpg)

>And who is Lee Tumpenny ? Is he/she connected with Parliament ?

it appears that 'submitted by' is not a clear enough indication that annotations are added by users

**Comment by zarino** [comment]
How about something that makes it look more like a "comment" by a user?

![annotation-concept](https://cloud.githubusercontent.com/assets/739624/16763881/077cbe56-4823-11e6-9312-a88190f708fe.png)

Any thoughts, @wrightmartin @dracos?

**Comment by MyfanwyNixon** [comment]
Yes that's good! Especially 'TWFY user notes'. Could push it even further with a little blank persona/figure if you wanted to...

**Comment by wrightmartin** [comment]
Looks good, I think the tail on the speech mark is a little cutesy, but that's picking the tiniest of nits
